library/globals.inc.php

```diff
@@ -204,6 +204,7 @@ function gblTimeZones()
             xl('General Theme') .'*',
             'css',
             'style_light.css',
+            'style_dark.css',
```

Comment:
this won't work; this is just a setting for the default setting.
Are you trying to create a Dark option in the administration->globals:
https://github.com/openemr/openemr/blob/master/interface/super/edit_globals.php#L702-L751
style_dark.scss will be used by build to create a style_dark.css in public/themes, so it should show up there as an option

Reply:
I'll write this in my next commit so I don't make the same mistake here 😂